PagerDuty/pagerduty.30s.rb

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/ruby
+# <bitbar.title>PagerDuty</bitbar.title>
+# <bitbar.version>v1.0</bitbar.version>
+# <bitbar.a uthor>Allan Frese</bitbar.author>
+# <bitbar.author.github>frese</bitbar.author.github>
+# <bitbar.desc>Shows current pagerduty alert status.</bitbar.desc>
+# <bitbar.dependencies>ruby (httparty gem)</bitbar.dependencies>
+
+require "json"
+require "pp"
+require "optparse"
+require "httparty"
+require "date"
+
+#--------------------------------------------------------------------
+# Create a configuration file ".pagerduty" in your HOME directory
+# TOKEN: <pagerduty-token>
+# DOMAIN: <your-pagerduty-domain>
+# USERID: <your-pager-duty-user-id>
+#--------------------------------------------------------------------
+
+class PagerDuty
+
+    def main
+        begin
+            # Init some defaults
+            $verbose = false
+            $id      = nil
+            $command = "GET"
+            $color   = Hash.new
+            begin
+                $config  = YAML.load(File.read("#{ENV['HOME']}/.pagerduty"))
```

Comment:
Convention with our other plugins has been to use environment variables for configuration. This is a bit of an unsolved problem right now - but I'm not sure plugins should be reading config from disk just yet. Could you please change these to environment variables instead?